### DISCUSSION TITLE
Event with multiple dates that can only be booked as a package

### DISCUSSION THREAD

**Discussion by leav94** [pretix/pretix#1947] [Ideas]
It would be helpful to be able to create an event with multiple dates that can be booked as a package. Then there should be a list that shows all dates belonging to the event. In addition, these dates must be exportable to one's own calendar. 

So far, only the time period of an event with several dates can be specified, so that the calendar function does not work or is not useful. Or you can create an event as a series, but then all individual dates would have to be booked seperately and not the dates as a package. 

I would be happy if you implement a solution. Thank you!

**Comment by gilbertf** [comment]
Similar requirement here. We (non profit) offer courses that, e.g. last for 10 sessions and can be very nicely presented to the user as event series. Since we require the users to attend all events of a series we are currently disabling the possibility to book the events, except the first one of a series:
![grafik](https://user-images.githubusercontent.com/222388/161932921-b4cfaf30-22b4-485e-a9c2-75e40a972029.png)
From user experience this displays nicely in the calendar. Unfortunatly from event setup perspective its quite complicated and error prone.

I would expect this would be a important feature to extend the possible application of pretix, since you will find a lot of events with fixed number of dates.

**Comment by raphaelm** [comment]
I believe either "cross-date packages" or "multi-timeframe dates" will happen eventually, but it might be a while since these will require some very complex and hard design choices.
